examples/todoapp/test/feature_statistics/blocs/statistics_test.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/annotations.dart';
+import 'package:mockito/mockito.dart';
+import 'package:rx_bloc/rx_bloc.dart';
+import 'package:rx_bloc_test/rx_bloc_test.dart';
+import 'package:todoapp/base/common_blocs/coordinator_bloc.dart';
+import 'package:todoapp/feature_statistics/blocs/statistics_bloc.dart';
+import 'package:todoapp/feature_statistics/models/todo_stats_model.dart';
+import 'package:todoapp/feature_statistics/services/statistics_service.dart';
+
+import '../../Stubs.dart';
+import 'statistics_test.mocks.dart';
+
+@GenerateMocks([
+  CoordinatorBlocType,
+  CoordinatorEvents,
+  CoordinatorStates,
+  StatisticsService,
+])
+void main() {
+  late CoordinatorBlocType _coordinatorBloc;
+  late CoordinatorEvents _coordinatorEvents;
+  late CoordinatorStates _coordinatorStates;
+  late StatisticsService _service;
+
+  void _defineWhen() {
+    when(_coordinatorStates.onTodoListChanged)
```

Comment:
For the sake of flexibility you can pass the data as arguments
 - Stubs.todoList
 - Stubs.todoListStatistics